lti_consumer/lti_1p3/consumer.py

```diff
@@ -417,6 +421,12 @@ def _validate_preflight_response(self, response):
         :param response: the preflight response to be validated
         """
         try:
+            # TODO: Per the specification, I believe we're meant to also do the following.
+            # 1. validate that the learner represented by "login_hint" is the currently authenticated user
+            #          making this request.
+            # 2. confirm that the redirect_uri matches one of the registered launch endpoints for the Tool
+            #          integration identified by the client_id, as specified during the configuration phase.
+            # TODO: Check in with Giovanni.
```

Comment:
@MichaelRoytman 
> validate that the learner represented by "login_hint" is the currently authenticated user

On the LTI consumer, we set `login_hint` as the [block id](https://github.com/openedx/xblock-lti-consumer/blob/master/lti_consumer/lti_xblock.py#L1516) - ideally, we should keep this consistent, but I'm ok to change on both places.

> 2. confirm that the redirect_uri matches one of the registered launch endpoints for the Tool

We did that initially, but a bunch of tools complained or stopped working. When using deep linking, the content URL changes and it would be a bit complicated to verify that here if the consumer is being instanced without looking at the stored deep linking configurations.

See: https://github.com/openedx/xblock-lti-consumer/blame/b80ca5fdecae89ba3857c71d8b86aa7215b42e6b/lti_consumer/lti_1p3/consumer.py#L403

Reply:
Okay, I'll drop a comment about `redirect_uri`. For `login_hint`, perhaps we'll deal with that when we do the refactor of the library to be less dependent on xBlocks. I don't want to mess with it right now.